app/views/admin/topical_event_organisations/index.html.erb

```diff
@@ -12,7 +12,7 @@
 
 <div class="govuk-!-margin-bottom-8">
   <%= render "components/secondary_navigation", {
-    aria_label: "Document tabs",
+    aria_label: "Document navigation",
```

Comment:
This is a pre-existing issue but it looks like this was copy pasted and should be `"Topical Events navigation"`